.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
can we do this from a locally installed binary rather than using `npx` here? that would mean defining lint-staged as a dependency in a package.json somewhere

Reply:
updated to `yarn lint-staged`